src/main/java/ParkingLot/Admin.java

```diff
@@ -0,0 +1,20 @@
+package com.company;
```

Comment:
how come you have package name com.company here?
this class is in the ParkingLot package

Reply:
done